bedrock/settings/static_media.py

```diff
@@ -403,6 +403,18 @@
         ),
         'output_filename': 'css/firefox_mobile_download.css',
     },
+    'firefox_mobile_download_desktop': {
+        'source_filenames': (
+            'css/sandstone/sandstone-resp.less',
+            'css/tabzilla/tabzilla-static.less',
+            'css/base/simple_footer.less',
+            'css/base/send-to-device.less',
+            'css/firefox/new/horizon/common.less',
+            'css/firefox/new/horizon/scene1.less',
```

Comment:
If we're using this horizon design more then maybe we should split it out in it's own file (not in this PR, but a separate bug). But for now, rather than include all the CSS from `/firefox/new/`, could we take only what's needed?